spec/models/spree/adjustment_spec.rb

```diff
@@ -491,5 +490,21 @@ module Spree
         end
       end
     end
+
+    context "return authorization adjustments" do
+      let!(:return_authorization) { create(:return_authorization, amount: 123) }
+      let(:order) { return_authorization.order }
+      let!(:return_adjustment) {
+        create(:adjustment, originator: return_authorization, order: order,
+                            adjustable: order, amount: 456)
+      }
+
+      describe "updating the adjustment" do
+        it "sets a negative value equal to the return authorization amount" do
+          return_adjustment.update!
+          expect(return_adjustment.reload.amount).to eq(-1 * return_authorization.amount)
```

Comment:
At the risk of being annoying, I would probably write:
```suggestion
          expect { return_adjustment.update! }.
            to change { return_adjustment.reload.amount }.to(-123)
```

I don't trust myself to make a mistake writing `-1 * amount` and then testing the same incorrect result in the spec. The expect block syntax is more complicated, difficult to read but catches mistakes in the test setup.